go/cmd/dolt/commands/sqlserver/servercontroller.go

```diff
@@ -19,8 +19,10 @@ import (
 )
 
 type ServerController struct {
```

Comment:
The way this was implemented was deadlocking some of the tests.  Specifically TestServerBadArgs would hang on the call to WaitForStart as the server never started and exited early.  Now if the server is stopped without ever being started WaitForStart will return.